src/js/src/components/SearchSingleItemComponents/SearchSingleItemFocusImage.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="searchSingleItemFocusImage" @click="closeImage">
+    <img :src="image" @click="stopCloseImage">
+    <button @click="closeImage">
```

Comment:
You could try @click.stop here and see if you can do without stopCloseImage() function.

Reply:
Fixed.